rpc/transaction_test.go

```diff
@@ -649,28 +647,29 @@ func TestTransactionReceiptByHashV0_6(t *testing.T) {
-			"execution_status": "REVERTED",
-			"finality_status": "ACCEPTED_ON_L2",
-			"block_hash": "0x76e0229fd0c36dda2ee7905f7e4c9b3ebb78d98c4bfab550bcb3a03bf859a6",
-			"block_number": 304740,
-			"messages_sent": [],
-			"events": [],
-			"revert_reason": "Error in the called contract (0x00b1461de04c6a1aa3375bdf9b7723a8779c082ffe21311d683a0b15c078b5dc):\nError at pc=0:25:\nGot an exception while executing a hint.\nCairo traceback (most recent call last):\nUnknown location (pc=0:731)\nUnknown location (pc=0:677)\nUnknown location (pc=0:291)\nUnknown location (pc=0:314)\n\nError in the called contract (0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7):\nError at pc=0:104:\nGot an exception while executing a hint.\nCairo traceback (most recent call last):\nUnknown location (pc=0:1678)\nUnknown location (pc=0:1664)\n\nError in the called contract (0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7):\nError at pc=0:6:\nGot an exception while executing a hint: Assertion failed, 0 % 0x800000000000011000000000000000000000000000000000000000000000001 is equal to 0\nCairo traceback (most recent call last):\nUnknown location (pc=0:1238)\nUnknown location (pc=0:1215)\nUnknown location (pc=0:836)\n",
-			"execution_resources":{"steps":0}
+			"type":"INVOKE",
+			"transaction_hash":"0xccd396d42a911917c510808d372b58ba937168b19e1254ac12aabe8f0db6b5",
+			"actual_fee":{"amount":"0x9bd5966b4619","unit":"WEI"},
+			"execution_status":"REVERTED",
+			"finality_status":"ACCEPTED_ON_L2",
+			"block_hash":"0x852225462ef12fe1955e473beb93877a8c9723650c56a13888057e2af184d3",
+			"block_number":19957,
+			"messages_sent":[],
+			"events":[{"from_address":"0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7","keys":["0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9"],"data":["0x37a10f2808c05f4a328bdac9a9344358547ae4676ebddc005e24ff887b188fd","0x1176a1bd84444c89232ec27754698e5d2e7e1a7f1539f12027f28b23ec9f3d8","0x9bd5966b4619","0x0"]}],
+			"revert_reason":"Error in the called contract (0x037a10f2808c05f4a328bdac9a9344358547ae4676ebddc005e24ff887b188fd):\nError at pc=0:4273:\nGot an exception while executing a hint: Custom Hint Error: Execution failed. Failure reason: 0x54696d657374616d7020697320696e2074686520667574757265 ('Timestamp is in the future').\nCairo traceback (most recent call last):\nUnknown location (pc=0:67)\nUnknown location (pc=0:1997)\nUnknown location (pc=0:2727)\nUnknown location (pc=0:3582)\n",
+			"execution_resources":{"steps":16849,"pedersen_builtin_applications":16,"range_check_builtin_applications":174,"ec_op_builtin_applications":3}
 		}`
 
-		integClient := feeder.NewTestClient(t, &utils.Integration)
+		integClient := feeder.NewTestClient(t, &utils.Sepolia)
 		integGw := adaptfeeder.New(integClient)
 
-		blockWithRevertedTxn, err := integGw.BlockByNumber(context.Background(), 304740)
+		blockWithRevertedTxn, err := integGw.BlockByNumber(context.Background(), 19957)
```

Comment:
I've found no difference in feeder response either from old & recents blocks.
So for these 2 new tests "reverted" & "v3 tx" I'm using the same block `19957` here 
(v0.6 api) and in the [following test](https://github.com/NethermindEth/juno/pull/1880/files#diff-92b1f47d3a0b6c346cb6e8bf24d9d0ac8b37a1c80adaa5e74922f9ab358b2ef5R715)